src/Components/Locations.js

```diff
@@ -0,0 +1,61 @@
+// "https://ghibliapi.herokuapp.com/locations/"
+import axios from "axios";
+import { Component } from "react";
+
+class Locations extends Component {
+  state = { places: [], showLoca: false };
+
+  fetchLocations = async () => {
+    try {
+      const res = await axios.get("https://ghibliapi.herokuapp.com/locations/");
+      this.setState({ places: res.data.map((location) => {
+          location = {name: location.name, climate: location.climate, terrain: location.terrain}
+      }) });
+        // debugger
+    } catch (error) {
+      this.setState({ places: [] });
+    }
+  };
+
+  componentDidMount() {
+    this.fetchLocations();
+  }
+ 
+
+  handleClick = (e) => {
+    // this.setState((prevState) => {
+    //   return { showLoca: !prevState.showLoca };
+    // });
+    this.setState({showLoca: !this.state.showLoca})
+  };
+
+  render() {
+    const { places, showLoca } = this.state;
+    const allLocations =  places.map((placesObj) => {
+        // <li>
+        <p style={{ color: "yellow" }}> Name: {placesObj.name} </p>
+        {/* <p style={{ color: "yellow" }}> Climate: {placesObj.climate} </p> */}
+        {/* <p style={{ color: "yellow" }}> Terrain: {placesObj.terrain} </p> */}
+        // </li>
```

Comment:
You need these commented out lines. But you should also be checking text for climate and terrain. 
Something like
 locationObj.climate === 'TODO' ? "Unknown" : locationObj.climate